server/workers/userCreated.js

```diff
@@ -0,0 +1,28 @@
+import {connect} from 'src/db'
+import {findContactByEmails, addIdmToCrm} from 'src/server/actions/crm'
+import createMember from 'src/server/actions/createMember'
+
+const r = connect()
+
+export function start() {
+  const jobService = require('src/server/services/jobService')
+  jobService.processJobs('userCreated', processUserCreated)
+}
+
+export async function processUserCreated(idmUser) {
+  try {
+    const contactVid = (await findContactByEmails(idmUser.emails)).vid
+
+    console.log(`CRM Match Found: Syncing IDM user ${idmUser.id} with CRM contact ${contactVid}...`)
+
+    await r.table('users')
+      .get(idmUser.id)
+      .update({hubspotContactId: contactVid})
+    await addIdmToCrm(idmUser.id, contactVid)
```

Comment:
Let's make sure that in all cases, the member is created. Either put hubspot stuff last or wrap in try/catch and log error but don't throw.